src/plugin/reflection_class.cpp

```diff
@@ -65,8 +65,33 @@ namespace chapel {
       }
     }
 
+    // write grand chindren: Message nested types
+    // handled here since nested records and enum declaration in records
+    // is not yet supported in Chapel.
```

Comment:
"is not" -> "are not", you are talking about two categories so it should be plural

Reply:
Yes, I will correct it :sweat_smile: